### ISSUE TITLE
Pytoch - > onnx - > tensorflow ，However, when using tensorflow for inference, an error occured!!!

### ISSUE THREAD

**Issue by feifeiwei** [onnx/onnx-tensorflow#579]
**Describe the bug**

InvalidArgumentError: ValueError: callback pyfunc_0 is not found
Traceback (most recent call last):

**To Reproduce**

**ONNX model file**

If applicable, attach the onnx model file in question using Gist, DropBox or Google Drive.

**Python, ONNX, ONNX-TF, Tensorflow version**

This section can be obtained by running `get_version.py` from util folder.
 - Python version: 3.7.3
 - ONNX version: 1.6.0
 - ONNX-TF version:1.5.0
 - Tensorflow version: 1.14.0

**Additional context**

**Comment by chinhuang007** [comment]
Does the error mention which operator caused it? Is it possible to share the onnx file for debugging?

**Comment by feifeiwei** [comment]
## python onnx2tf.py
```
import onnx
import warnings
from onnx_tf.backend import prepare

warnings.filterwarnings('ignore') # Ignore all the warning messages in this tutorial
model = onnx.load('./weights/lane_weights.onnx') # Load the ONNX file
tf_rep = prepare(model) # Import the ONNX model to Tensorflow


print("**"*9)
print(tf_rep.inputs) # Input nodes to the model
print('-----')
print(tf_rep.outputs) # Output nodes from the model
print('-----')
print(tf_rep.tensor_dict) # All nodes in the model


tf_rep.export_graph('./weights/lane_weights.pb')
```

##python inference.py
```
#!/usr/bin/env python3
# -*- coding: utf-8 -*-

import cv2
from PIL import Image
import numpy as np
import tensorflow as tf
from utils.postprocess import embedding_post_process
from tensorflow.python.platform import gfile

'''
import onnx
import warnings
from onnx_tf.backend import prepare

warnings.filterwarnings('ignore') # Ignore all the warning messages in this tutorial
model = onnx.load('./weights/lane_weights.onnx') # Load the ONNX file
tf_rep = prepare(model) # Import the ONNX model to Tensorflow

'''

def get_img_np_nchw(filename):
    try:
        image = Image.open(filename).convert('RGB').resize((800, 288))
        miu = np.array([0.485, 0.456, 0.406])
        std = np.array([0.229, 0.224, 0.225])
        img_np = np.array(image, dtype=float) / 255.
        r = (img_np[:,:,0] - miu[0]) / std[0]
        g = (img_np[:,:,1] - miu[1]) / std[1]
        b = (img_np[:,:,2] - miu[2]) / std[2]
        img_np_t = np.array([r,g,b])
        img_np_nchw = np.expand_dims(img_np_t, axis=0)
        return img_np_nchw
    except:
        print("RuntimeError: get_img_np_nchw({}).".format(filename))



if __name__ == '__main__':
    
    filename = "./laneDemo/00020.jpg"
    tf_pb_path = './weights/lane_weights.pb'
    
    img_np_nchw = get_img_np_nchw(filename)
    
    with tf.Session() as sess:
        print("load graph")
        with gfile.FastGFile(tf_pb_path, 'rb') as f:
            graph_def = tf.GraphDef()
            graph_def.ParseFromString(f.read())
    
        sess.graph.as_default()
        tf.import_graph_def(graph_def, name='')
    
        inp  = sess.graph.get_tensor_by_name('actual_input:0')
        out1 = sess.graph.get_tensor_by_name('output1:0')
        out2 = sess.graph.get_tensor_by_name('output2:0')

        out  = [out1,out2]
        feed_dict = {inp: img_np_nchw}
    
        result1,result2 = sess.run(out,feed_dict)
  
        print(result1.shape)
        print(result2.shape)

```

the onnx file cound be found

[lane_weights.onnx.zip](https://github.com/onnx/onnx-tensorflow/files/4233847/lane_weights.onnx.zip)

**Comment by sdmonov** [comment]
@feifeiwei the problem is in the first MaxPool. This onnx file uses v9 of the opset. The implementation for this operator for v9 does not  support pads which cannot match SAME paddings in Tensorflow. In this case MaxPool is used with kernel_shape=[3,3], strides=[2,2] and pads=[1,1,1,1]. As tensorflow max_pool op does not support explicit padding and the operator implementation will try to check if the provided pads match SAME paddings in tensorflow ([0,0,1,1] in our case) and use "SAME" as paddings for the tf.nn.max_pool op. In case pads does not match SAME padding the code will use py_func and python implementation of MaxPool. This py_func does not serialize in the exported pb file and during inference will fail. There is a new implementation of the MaxPool op for v10 which fully supports MaxPool without using py_func. Also the PR #565 removes the old code and uses the new implementation for all versions of the opset. Once it is merged you should not see the problem. 

You can try test this PR. Another option is instead of exporting to pb file you can directly run the prepared model with tf_rep.run()

**Comment by feifeiwei** [comment]
Okay,  i will try it and report soon.

**Comment by JourneyGo** [comment]
I have found the same error. Is this #565 PR useable for 1.x branch?

**Comment by mgarbade** [comment]
- There is a fully working conversion example in google Colab as mentioned in [this](https://stackoverflow.com/a/58576060/2191652) stackoverflow answer
- however the converted model only works on CPU (mobile phone and computer)
but not on GPU (mobile phone)
-  also the CPU version is much slower than a corresponding model build in TF directly as mentioned [here](https://stackoverflow.com/a/62343410/2191652)

**Comment by sdmonov** [comment]
@JourneyGo there is new PR #674  that backports the latest avg/max pool ops into the tf-1.x.
